### ISSUE TITLE
(M) As a registered user, I want to disable my account, so that I can act within my GDPR rights.

### ISSUE THREAD

**Issue by sfinn85** [fedora-infra/noggin#39]
AC:
We have a SAR script to list information about users in our repository
We have a means to link users to the pagure instance where they can request an account disable

DoD:
All prs implementing the above are merged and have been verified

**Comment by relrod** [comment]
A few notes for this:

* Most of this work will happen in the infra GDPR/SAR scripts/playbooks. They need to be updated for IPA.
* The securitas UI should probably link to the pagure tracker for filing GDPR/SAR requests -- making this self-service isn't a goal, but communicating how to accomplish it is.

**Comment by sfinn85** [comment]
HI @relrod 
I don't have an option to pull this user story on to the AAA board into the to do lane. Can you take a look to see if you can do this?

**Comment by ryanlerch** [comment]
> * The securitas UI should probably link to the pagure tracker for filing GDPR/SAR requests -- making this self-service isn't a goal, but communicating how to accomplish it is.

Just out of interest, can a user delete or disable an account with freeipa (not within the context of a GDPR request) ixs this something we need to do in the UI

**Comment by relrod** [comment]
@ryanlerch No, for various reasons we don't want people to be able to delete accounts (just like current FAS). This ticket is mainly a tracking ticket just to note that and also as a reminder that we'll need to update our GDPR/SAR playbooks in infra to deal with IPA instead of FAS when a request comes in. But on the portal side, we don't need to do anything here.

**Comment by StephenCoady** [comment]
So just to be clear before I start working on this. The acceptance criteria is:

* Update the relevant playbooks to allow the removal of accounts in IPA
* Create a new view which just points to the pagure tracker for creating a ticket to remove an account

Sound OK @relrod @sfinn85 ?

**Comment by StephenCoady** [comment]
It also looks like there may be a need to update some SOPs related to removing accounts, even if it is just rewording to move away from FAS.

Specifically here: https://fedora-infra-docs.readthedocs.io/en/latest/sysadmin-guide/sops/gdpr_delete.html

**Comment by relrod** [comment]
@StephenCoady correct. I'd say "Create a view" is optional (current FAS doesn't have one), but if you feel like working on an easyfix issue, feel free to add it.

**Comment by sfinn85** [comment]
Playbook query @StephenCoady following up with Kevin . It is a small patch if required - but there are doubts now as to whether or not we can actually delete accounts.

**Comment by sfinn85** [comment]
Sprint 3 planning update: (M) update PR to use Ryan's new style , create script ( admin print out user details when requested) change to SOP's for Sys Admin - query. Solution: Create PR add note - do not merge until 3/31/20. Name query -  Sarah to start mail thread.

**Comment by sfinn85** [comment]
Hi Stephen, do we need to change the title to ' disable' instead of 'delete'? 

**Acceptance Criteria:**

- PR updated to Ryan's new style
- Script created to allow admin print out user details when requested
- Create PR and add note for System admin to change SOP when we merge on 3/31/20. 

**Definition of done:**

- All AC is met
- Tests ran regarding Script
- Link added to this story for PR created for System team to action on 3/31/20

**Comment by StephenCoady** [comment]
Update on this. I have two PRs which I think pretty much cover this application's requirements.

https://github.com/fedora-infra/securitas/pull/106
https://github.com/fedora-infra/securitas/pull/97

The only thing left is to update the SOPs. I'm starting to second guess whether that should be part of this user story or whether it should be part of the ansible install user story we are inevitably going to have to do for the system admins to install this application. It makes sense to do it there instead as variables and app names will need to be referred to in the SOP. Open to comment.

**Comment by sfinn85** [comment]
Daily stand up update:

Query regarding SOP - SOP to be created as a different issue to be worked on later. 

Just for my clarity, do we need to change the AC and DOD to change this.  If the PR is to be logged later on?

@StephenCoady can you add link to the new issue you are creating here just so if we refer back to this user story at any point, we have the whole story. 

Thanks a mill

Sarah

**Comment by StephenCoady** [comment]
@sfinn85 I'll add some acceptance criteria to the main description as its actually empty at the moment. 

The newly created issue is linked just above your comment.
